src/libs/OptionsListUtils.js

```diff
@@ -479,7 +479,7 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
         reportName = ReportUtils.getReportName(report);
     } else {
         reportName = ReportUtils.getDisplayNameForParticipant(accountIDs[0]);
-        result.keyForList = accountIDs[0];
+        result.keyForList = String(accountIDs[0]);
```

Comment:
Does the key need to be string? Or I thought we decided that the accountIDs will be ints

Reply:
This is just used internally for react, and it's supposed to receive string (in `optionPropTypes`)

Reply:
yeaaaaaaa accountIDs will be ints, but a few components assume the `keyForList` prop is string, if i remember correctly